meteor-app/imports/ui/pages/for-employers/for-employers.jsx

```diff
@@ -13,12 +13,32 @@ const T = i18n.createComponent(t);
 
 const vizeBlue = "#2699FB";
 const vizePaleBlue = "#F1F9FF";
+const horizontalPaddingVal = "15px";
+
+const sizes = {
+	desktop: 992,
+	tablet: 768,
+	phone: 576,
+};
+
+const media = Object.keys(sizes).reduce((acc, label) => {
+	acc[label] = (...args) => css`
+		@media (max-width: ${sizes[label] / 16}em) {
+			${css(...args)}
+		}
+	`;
+
+	return acc;
+}, {});
 
 const P = styled.p`
```

Comment:
Defining `styled-components` like this in the component itself will make the component code stale after some time. The component code should be only about what the component do functionally not how it looks, it's is similar to CSS, we don't define CSS rules in the component JS file, we create a new file for that because it has a different purpose. I recommend we create a new file called `styles.js` in the same directory level as the component, in it you can define the styles for the component's elements, export and import it to be used in the component.

@Mahfouzer Please wait for @julianjear approval on this change before you do it.

Reply:
I thought the same.
 once @julianjear approves I will create the config file of variables and sizes and do the mentioned change.

Reply:
We are currently in the process of converting to `styled-components`. We are planing on organizing the code into small reusable components instead of splitting the HTML/CSS/JS. I am aware that having the styles in the same file as the logic of such a large component is poor quality code, but I believe the problem lies in having large monolithic components, not in having logic and styles mixed.